_config.yml

```diff
@@ -315,9 +309,6 @@ post_navigation: left
 # See: https://theme-next.org/docs/theme-settings/custom-pages
 # ---------------------------------------------------------------
 
-# Enable "cheers" for archive page.
```

Comment:
Why remove?

Reply:
Yes. Always enabled.

Reply:
Then it would be very nice (user friendly) to write a simple guide in this PR to tell users **how**.

Reply:
**how**: https://theme-next.org/docs/advanced-settings.html#Injects